deepomatic/api/resources/task.py

```diff
@@ -62,17 +78,14 @@ def list(self, task_ids):
         assert(isinstance(task_ids, list))
         return super(Task, self).list(task_ids=task_ids)
 
-    def wait(self, timeout=60, wait_exp_multiplier=0.05, wait_exp_max=1.0):
+    def wait(self, **retry_kwargs):
         """
         Wait until task is completed. Expires after 'timeout' seconds.
         """
         try:
-            retryer = Retrying(wait=wait_random_exponential(multiplier=wait_exp_multiplier, max=wait_exp_max),
-                               stop=stop_after_delay(timeout),
-                               retry=retry_if_result(is_pending_status),
-                               before=before_log(logger, logging.DEBUG),
-                               after=after_log(logger, logging.DEBUG))
-            retryer(self._refresh_status)
+            retry_get_tasks(self._refresh_status,
+                            retry_if_result(is_pending_status),
+                            **retry_kwargs)
         except RetryError:
             raise TaskTimeout(self.data())
```

Comment:
maybe add more info to the exception:
- timeout duration?
- total wait time?
- if error when getting task: say we got an error getting task? and which error?

Reply:
I store it now in TaskTimeout.retry_error